website/docs/Installation.md

```diff
@@ -118,6 +118,17 @@ Example notebooks:
 [Automated Code Generation and Question Answering with Qdrant based Retrieval Augmented Agents](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_qdrant_RetrieveChat.ipynb)
 
 
+- #### TeachableAgent
+
+To use TeachableAgent, please install AutoGen with the [teachable] option.
+```bash
+pip install "pyautogen[teachable]"
```

Comment:
@thinkall would you like to update line 98-100 about the version supported for retrievechat?

Reply:
@thinkall, after reading this message, can you resolve it so as to not block this PR? Thanks.

Reply:
> @thinkall, after reading this message, can you resolve it so as to not block this PR? Thanks.

Hi @rickyloynd-microsoft , I’m currently afk and will be back in a few hours. Could you help me update the version? Just remove the “<0.2.0” condition should be good. @sonichi , is it OK to you? Thanks.